pkg/generators/matrix_test.go

```diff
@@ -95,14 +122,14 @@ func TestMatrixGenerate(t *testing.T) {
 		},
 	}
 
-	for _, c := range testCases {
-		cc := c
+	for _, testCase := range testCases {
+		testCaseCopy := testCase // Since tests may run in parallel
```

Comment:
Test cases won't run in parallel unless you call `t.Parallel()` explicitly from your test code which is something that I always do to make sure the tests executed faster.

Reply:
Will change to parallel then!